src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html

```diff
@@ -20,7 +20,7 @@ <h2 i18n>rundown-overview-component.no-rundowns-available.message</h2>
       <ng-container matColumnDef="status">
         <mat-header-cell *matHeaderCellDef></mat-header-cell>
         <mat-cell *matCellDef="let basicRundown" >
-          <sofie-pulsating-dot *ngIf="basicRundown.isActive" [color]="Color.ON_AIR" [animationDurationSeconds]="3"></sofie-pulsating-dot>
+          <sofie-pulsating-dot *ngIf="basicRundown.mode === RundownMode.ACTIVE" [color]="Color.ON_AIR" [animationDurationSeconds]="3"></sofie-pulsating-dot>
```

Comment:
I assume that having a differently styled indicator for rehearsal is not part of this task. But I think that the pulsating dot should show as is when in rehearsal mode.

Reply:
That is something to be handled in SOF-1568, but it is an acceptance criteria of the story. However, I agree with Rasmus, it should not be showing the ON_AIR color
![image](https://github.com/tv2/ng-sofie/assets/23049647/21eb7e46-57bb-4b45-b5d9-489ff6d80160)